src/frontend/komponenter/Behandling/Brev/Brev.tsx

```diff
@@ -41,10 +42,20 @@ const Brev: React.FC<Props> = ({ behandlingId }) => {
         });
     };
 
+    const hentBrev = () => {
+        axiosRequest<string, any>({
+            method: 'GET',
+            url: `/familie-ef-sak/api/brev/${behandlingId}`,
+        }).then((respons: Ressurs<string>) => {
```

Comment:
Bør kanskje håndtere at brevet ikke finnes/evt. ikke tilgang?

Reply:
Ja, i første omgang la vi den til for å kunne teste. Etter hvert skal bare denne dukke opp under steg 2 av totrinnskontrollen (og da vil vel "Generer brev" bare dukke opp på steg 1), men nå vises begge knapper hele tiden.